src/libstd/time/duration.rs

```diff
@@ -290,7 +290,7 @@ impl Duration {
     }
 
     /// Checked `Duration` subtraction. Computes `self - other`, returning [`None`]
-    /// if the result would be negative or if overflow occurred.
+    /// if underflow or overflow occurred.
```

Comment:
Please revert these changes. The term "overflow" is correct, see #46898.

Reply:
:+1: